### ISSUE TITLE
[Bukkit] {display_name} is not actually Displayname

### ISSUE THREAD

**Issue by burdoto** [Mineaurion/AurionChat#26]
The `{display_name}` placeholder does not actually insert the display name, but instead an unformatted version of the username

Through EssentialsX' `/nick` command, I set myself a nickname:
![grafik](https://github.com/Mineaurion/Aurionchat/assets/42473485/0c44f35b-3af4-4281-9279-da2a68107bab)

However, its not actually supplied by AurionChat:
![grafik](https://github.com/Mineaurion/Aurionchat/assets/42473485/17dc7376-6fd9-4124-978f-5850d763a454)

Additionally, AurionChat requires me to supply the `{prefix}` and `{suffix}` placeholders manually; while most other plugins that use a displayname placeholder grab the entire name from essentials or whatever else; in which the prefix, suffix and nickname are automatically included.

This could be resolved by implementing Placeholder API, as that would provide ways to obtain Essentials- and other displaynames easily.

**Comment by Yann151924** [comment]
You speak about bukkit platform ? Just to clarify :)

**Comment by burdoto** [comment]
yes, sorry for not specifying. this is from the bukkit version of aurionchat
